pkg/manifests/manifests.go

```diff
@@ -291,22 +296,103 @@ func DaemonSet(component string, plat platform.Platform, namespace string) (*app
 		return nil, fmt.Errorf("unexpected type, got %t", obj)
 	}
 
+	hostPathDirectory := corev1.HostPathDirectory
+	hostPathSocket := corev1.HostPathSocket
+	ds.Spec.Template.Spec.Volumes = []corev1.Volume{
+		{
+			// needed to get the CPU, PCI devices and memory information
+			Name: rteSysVolumeName,
+			VolumeSource: corev1.VolumeSource{
+				HostPath: &corev1.HostPathVolumeSource{
+					Path: "/sys",
+					Type: &hostPathDirectory,
+				},
+			},
+		},
+		{
+			Name: rtePodresourcesSocketVolumeName,
+			VolumeSource: corev1.VolumeSource{
+				HostPath: &corev1.HostPathVolumeSource{
+					Path: "/var/lib/kubelet/pod-resources/kubelet.sock",
+					Type: &hostPathSocket,
+				},
+			},
+		},
+	}
+
+	containerPodResourcesSocket := fmt.Sprintf("/%s/kubelet.sock", rtePodresourcesSocketVolumeName)
```

Comment:
let's use `filepath.Join` please

Reply:
here and in all the similar places (if any)

Reply:
done